test/metatx/ERC2771Forwarder.t.sol

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.19;
+
+import "forge-std/Test.sol";
+import "contracts/metatx/ERC2771Forwarder.sol";
+import "contracts/mocks/CallReceiverMock.sol";
+
+struct ForwardRequest {
+    address from;
+    address to;
+    uint256 value;
+    uint256 gas;
+    uint256 nonce;
+    uint48 deadline;
+    bytes data;
+}
+
+contract ERC2771ForwarderMock is ERC2771Forwarder {
+    constructor(string memory name) ERC2771Forwarder(name) {}
+
+    bytes32 private constant _FORWARD_REQUEST_TYPEHASH =
```

Comment:
This is already declared in the `ERC2771Forwarder`.

I wonder if constant and immutable variable should be internal instead of private. There is no risk of writting to it.

Reply:
I'd be in favor of making the `_FORWARD_REQUEST_TYPEHASH` internal, but I'm also in favor of having an internal `_structHash`, what do you think about both? 

CC @frangio because it's a design decision.

Reply:
Internal seems fine.